maestrowf/__init__.py

```diff
@@ -264,6 +272,67 @@ def render_to_str(self, theme=None, width=200):
         return _printer.file.getvalue()
 
 
+class SummaryStatusRenderer(FlatStatusRenderer):
+    """Flat, simple summary table layout"""
+
+    layout_type = "summary"        # Defines name in factory/cli
+
+    def update_theme(self):
+        self._theme_dict["Step Prefix"] = "blue"
+        self._theme_dict["State"] = "bold red"
+        self._theme_dict["Count"] = "blue"
+
+    def process_status_data(self, status_data):
+        """Construct the summary dictionary"""
+        summary_data = OrderedDict()
```

Comment:
As an FYI, we've dropped support for Python versions < 3.6 -- fortunately, as of 3.6 the standard library dictionary underwent performance optimizations that cause it to function in the same way as an `OrderedDict` making this unnecessary.